src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -151,6 +151,55 @@ void CascadiaSettings::_CreateDefaultProfiles()
     _profiles.emplace_back(powershellProfile);
     _profiles.emplace_back(cmdProfile);
     
+    // Add all WSL distros from: HKCU\Software\Microsoft\Windows\CurrentVersion\Lxss
+    Profile WSLDistro{};
+    HKEY hLXSSKey;
+    // Should I use RegOpenCurrentUser to access the uncached profile?
+    // https://docs.microsoft.com/en-us/windows/desktop/api/winreg/nf-winreg-regopencurrentuser#remarks
+    if (RegOpenKeyExW(HKEY_CURRENT_USER, L"Software\\Microsoft\\Windows\\CurrentVersion\\Lxss",
+        0, KEY_READ, &hLXSSKey) == ERROR_SUCCESS)
+    {
+        std::filesystem::path AppsDirectory = ExpandEnvironmentVariableString(L"%ProgramFiles%");
+        AppsDirectory /= "WindowsApps";
+        std::filesystem::path WSLCommandLine = L"";
+
+        wchar_t WSLKey[MAX_PATH] = L""; // Registry key for a distinct WSL Distribution
+        wchar_t DistributionName[MAX_PATH] = L""; // The name to be displayed by the flyout list
+        wchar_t PackageFamilyName[MAX_PATH] = L""; // The distinct name for the distribution
+
+        // These tell the Registry functions how much space they have to work with when returning a value.
+        DWORD WSLKey_Length = MAX_PATH;

```

Comment:
In general, the Windows Terminal code uses the same style as the rest of Windows when it comes to naming variables.  In particular, I'd suggest a declaration and name that better fits the existing style would be something like:

DWORD cchWSLKey{std::size(WSLKey)};

The 'cch' here is special notation sometimes referred to as ["hungarian notation"](http://www.imm.dtu.dk/~alan/hungarian.html) used to indicate 'count of characters'.

Windows developers are used to relying on the name of the variable to tell them at a glance what the expected usage and related characteristics of a variable are in a concise fashion and so it would be helpful if the same could be applied to the new code here through this file.

Reply:
Updated.